### PR TITLE
Show package version in server UI sidebar

### PR DESCRIPTION
**Pull request by gravi2**
@patrickhulce I am raising this PR to address #145. 

Below is a quick overview of what has changed:
1.  I am making use of process.env.npm_package_version to make this change at the build time. I assume the `lerna version` lifecycle scripts will update the package.json  and the same will be picked by `parcel` in the `build` command.
2. JSX Changes in page-sidebar.jsx to add the siderbar footer and version elements. 
3. CSS changes in app.css and page-sidebar.css

Attaching few screenshots for your reference.

desktop

mobile

### CODE REVIEW

File: packages/server/src/ui/app.css [hunk 1]
```diff
@@ -97,6 +97,8 @@
 
   --page-sidebar-width: 260px;
   --page-sidebar-nav-item-height: 24px;
+  --page-sidebar-footer-font-size: 14px;
```
Comment (patrickhulce) [inline packages/server/src/ui/app.css]: let's not use a different font-size for this, how about `--subtext-font-size`?

File: packages/server/src/ui/app.css [hunk 2]
```diff
@@ -97,6 +97,8 @@
...
+  --page-sidebar-footer-margin: 25px;
```
Comment (patrickhulce) [inline packages/server/src/ui/app.css]: we try not to use bespoke spacing, we have `var(--base-spacing)` for margin and padding :)

File: packages/server/src/ui/layout/page-sidebar.jsx [hunk 3]
```diff
@@ -51,6 +51,9 @@ export const PageSidebar = props => {
           }}
         />
       </div>
+      <div className="page-sidebar__footer">
+        <div className="page-sidebar__version">Version {process.env.npm_package_version}</div>
```
Comment (patrickhulce) [inline packages/server/src/ui/layout/page-sidebar.jsx]:
```suggestion
        <div className="page-sidebar__version">
          {/* This environment variable is set by npm/yarn on any script command. */}
          v{process.env.npm_package_version}
         </div>
```

File: packages/server/src/ui/layout/page-sidebar.css [hunk 4]
```diff
@@ -69,3 +69,17 @@
...
+  width: 100%;
+  display: flex;
+  flex-direction: row;
+  justify-content: center;
+  margin: var(--page-sidebar-footer-margin) 0;
+  font-size: var(--page-sidebar-footer-font-size);
+}
+
+.page-sidebar__version {
+  color:sandybrown;
```
Comment (patrickhulce) [inline packages/server/src/ui/layout/page-sidebar.css]:
```suggestion
  color: var(--inverted-secondary-text-color);
```

File: packages/server/src/ui/layout/page-sidebar.css [hunk 5]
```diff
@@ -69,3 +69,17 @@
   color: var(--inverted-text-color);
   text-decoration: none;
 }
+
+.page-sidebar__footer {
```
Comment (patrickhulce) [inline packages/server/src/ui/layout/page-sidebar.css]:
```suggestion
.page-sidebar__footer {
  position: absolute;
  left: 0;
  right: 0;
  bottom: var(--base-spacing);
  text-align: center;
}
```

maybe something like that?

File: packages/server/src/ui/app.css (hunk 1 above)
Comment (gravi2) [reply packages/server/src/ui/app.css]: @patrickhulce I have accepted your CSS changes and I have removed --page-sidebar-footer-font-size as we don't need it any more.

File: packages/server/src/ui/app.css (hunk 2 above)
Comment (gravi2) [reply packages/server/src/ui/app.css]: @patrickhulce I have accepted your CSS changes and I have removed --page-sidebar-footer-maring as we don't need it any more.

**Comment by gravi2** [comment]
@patrickhulce Thanks for your review comments. I have addressed the review feedback and the same is now available in the PR. Here is the latest screenshot after review changes, for your reference. 
Screenshot from 2019-12-12 01-22-30

**Comment by gravi2** [comment]
@patrickhulce I have fixed the linting issue using the autofix (--fix). Though I din't quite like the lint fix it did :)

**Comment by patrickhulce** [comment]
>  Though I din't quite like the lint fix it did :)

Haha yeah that one doesn't make the most sense to me either, looks like there's an open bug in Prettier for it already. (https://github.com/prettier/prettier/issues/7132#issuecomment-565489232)


Thanks @gravi2!

**Pull request merged**